cloudevents/extensions/opcua.md

```diff
@@ -0,0 +1,102 @@
+# OPC UA
+
+This extension defines the mapping of OPC UA dataset to CloudEvents to allow seamless routing of OPC UA dataset messages via different protocols, it therefor provides an recommendation to map known MANDATORY and OPTIONAL attributes using other extensions as well as defines own extended attributes.
```

Comment:
Can you wrap things at 80 columns?

Reply:
s/therefor/therefore/
s/an recommendation/a recommendation/
s/defines own extended/defines its own extension/

Reply:
s/MANDATORY/REQUIRED/   I think

Reply:
Thanks, I reall like that you add the vim replace syntax :)

Reply:
LOL long live vi !